images/server/Containerfile

```diff
@@ -9,9 +9,9 @@ ARG SAMBACC_REPO=https://github.com/samba-in-kubernetes/sambacc
 RUN SAMBACC_DISTNAME=latest \
     /usr/local/bin/build.sh ${SAMBACC_VER} ${SAMBACC_REPO}
 
-FROM registry.fedoraproject.org/fedora:34
+FROM registry.fedoraproject.org/fedora:35
 ARG INSTALL_PACKAGES_FROM=default
-ARG SAMBA_SPECIFICS=
+ARG SAMBA_SPECIFICS=daemon_cli_debug_output
```

Comment:
Since we add **daemon_cli_debug_output** here in _Containerfile_ I think we can remove `--build-arg=SAMBA_SPECIFICS="daemon_cli_debug_output"` from **build-nightly-server** make target.